src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  --logo-width: 40px;
+  --logo-height: 40px;
+  --color-background: #00acdc;
```

Comment:
It's better to write variables inside the :root selector

Reply:
If I use the root selector, the 4 tests fail. The links move away from the center.